src/Uno.UI/UI/Xaml/Controls/TimePicker/TimePickerSelector.iOS.cs

```diff
@@ -74,7 +73,7 @@ private void SetPickerMinuteIncrement(int minuteIncrement)
 			}
 		}
 
-		private void SaveInitialTime() => _initialTime = _picker.Date;
+		private void SaveInitialTime() => _initialTime = _picker?.Date;
```

Comment:
Why is the `?` needed here?

Reply:
That was my first fix. Because without the null check it was giving the error preventing the flyout to open. 
Then I check the TimePicker and even opening the flyout it was not presenting the default value.. so I made the rest of the fix.

I will recheck where else is calling to SaveInitialTyme.. 

The real issue here is the implementation of the TimePicker how differs from DatePicker.  DatePicker sets the initial Time on Opening event and Time Picker on the internal Open..

TimePicket needs a rewrite to conform with how DatePicker works.. for now I will live it with the null check (because right now the TimePicker in 3.4 does not work) I with time rewrite the Time Picker.  You can assign the rewrite to me. What do you think?

Reply:
Btw. If you don't set the initial Date on datePicker it gets the default from each platforms default.. on iOS is datetime now. TimePicker on uwp es 12:00pm .. in iOS is DateTime.now..

DatePicker and TimePicker needs a full check

Reply:
Ok, sounds good :-) .

I know that there are definitely plans to rewrite the `TimePicker` to match UWP UX, but there will probably still be the option to use the native picker if needed. I will ask the team and let you know